src/WPF/WPF.Viewer/Samples/LocalServer/LocalServerGenerateElevationProfile/LocalServerGenerateElevationProfile.xaml

```diff
@@ -63,5 +63,28 @@
                            TextWrapping="WrapWithOverflow" />
             </StackPanel>
         </Border>
+        <Border HorizontalAlignment="Left" VerticalAlignment="Bottom" Style="{StaticResource BorderStyle}">
+            <StackPanel>
+                <TextBlock Grid.Row="0"
+                           Margin="0,5"
+                           Text="Set Local Server TempPath:" />
+                <TextBox x:Name="tempPathBox"
+                         Grid.Row="1"
+                         Margin="0,5" />
+                <Grid Margin="0,5">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <Button Grid.Column="0"
+                            Click="setTempPathButton_Click"
+                            Content="Set" />
+                    <Button Grid.Column="1"
```

Comment:
Minor visual nitpick: buttons are up against each other and would look nicer with a small gap between them.